README.md

```diff
@@ -75,7 +75,7 @@ Just run the following command from the meshviewer directory:
 grunt
 ```
 
-This will generate `build/` containing all required files.
+This will generate a folder `build/` that will contain all required files.
```

Comment:
I don't like the order of the words. I would prefer:
> generate a `build/` folder

I think the proposed spelling sounds very German.

If you want to keep the order of words, you can switch the `a` to a `the`.